Client/Frontend/Widgets/BlurTableViewController.swift

```diff
@@ -0,0 +1,182 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Storage
+
+class BlurTableViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    var profile: Profile!
+    var site: Site!
+    var actions: [BlurTableViewAction] = []
+    var tableView = UITableView()
+    lazy var tapRecognizer: UITapGestureRecognizer = {
+        let tapRecognizer = UITapGestureRecognizer()
+        tapRecognizer.addTarget(self, action: #selector(BlurTableViewController.dismiss(_:)))
+        tapRecognizer.numberOfTapsRequired = 1
+        tapRecognizer.cancelsTouchesInView = false
+        return tapRecognizer
+    }()
+
+    lazy var visualEffectView : UIVisualEffectView = {
+        let visualEffectView = UIVisualEffectView(effect: UIBlurEffect(style: .Light))
+        visualEffectView.frame = self.view.bounds
+        visualEffectView.alpha = 0.90
+        return visualEffectView
+    }()
+
+    lazy var bookmarkAction : BlurTableViewAction = {
```

Comment:
These actions should be defined outside of the menu. In ASPanel instead. That way we can easily reuse the menu in cases where we want to show a similar menu.